Legacy/PlayFab/Multiplayer.api.json

```diff
@@ -3267,6 +3267,66 @@
         }
       ]
     },
+    "ExternalFriendSources": {
+      "name": "ExternalFriendSources",
+      "className": "ExternalFriendSources",
+      "classNameSpace": "PlayFab.Multiplayer.Models",
+      "assembly": "PlayFab.Multiplayer.Models",
+      "isenum": true,
+      "enumvalues": [
+        {
+          "name": "None"
+        },
+        {
+          "name": "Steam"
+        },
+        {
+          "name": "Facebook"
+        },
+        {
+          "name": "SteamOrFacebook"
+        },
+        {
+          "name": "Xbox"
+        },
+        {
+          "name": "SteamOrXbox"
+        },
+        {
+          "name": "FacebookOrXbox"
+        },
+        {
+          "name": "SteamOrFacebookOrXbox"
+        },
+        {
+          "name": "Psn"
```

Comment:
PSN friend support is new right? dumb question but do PSN friend queries not require a token or auth method of some kind? I know steam and facebook don't but Xbox does and I wouldn't be terribly surprised if PSN did as well given they are a more locked down console platform like xbox (but it's equally possible that they don't)

trying to understand how we might in the future reflect this in the PFMultiplayer lobby and matchmaking API and if there'd be a need for a matching PsnToken field or something

Reply:
Good questions. Yes, PSN is new. I've added a service dev into our Teams conversation to get the details.